include/psa/crypto_values.h

```diff
@@ -480,12 +480,6 @@
  * Camellia block cipher. */
 #define PSA_KEY_TYPE_CAMELLIA                       ((psa_key_type_t)0x2403)
 
-/** Key for the RC4 stream cipher.
- *
- * Note that RC4 is weak and deprecated and should only be used in
- * legacy protocols. */
-#define PSA_KEY_TYPE_ARC4                           ((psa_key_type_t)0x2002)
```

Comment:
I'm not sure here: in theory, some people might want to provide a driver for RC4 and then they would need the algorithm identifier to be declared here. @gilles-peskine-arm @ronald-cron-arm wdyt? (Same question for the `PSA_WANT_xxx` macros, and `crypto_knowledge.py` below.)

Reply:
In principle removing the identifier from `crypto_values.h` doesn't block drivers from implementing it: drivers can implement algorithms that the core doesn't recognize. Drivers would have to declare their “proprietary” algorithms in a separate header. Removing support from `crypto_sizes.h` is a problem though, something we want to resolve eventually.

I doubt that any of the algorithms we're removing are relevant — if they were we wouldn't be removing them. So feel free to remove them. We can easily add them back if needed.

Reply:
Thanks!